### PR TITLE
Add config attribute to allow skipping fuzzers for a benchmark.

### PR DESCRIPTION
**Pull request by inferno-chromium**
This is needed for KLEE and AFLCC which do not support all
benchmarks, but we still need to support building the rest
and running an experiment without constant build retries.

### CODE REVIEW

**Review by laurentsimon — APPROVED** [review]

**Comment by inferno-chromium** [comment]
Since fuzzer does not config file, i thought to just use benchmark config here. it feels weird to add fuzzer config just for this one attribute, plus might be better to think in terms of benchmarks, that which fuzzer don't work with it

File: experiment/build/builder.py [hunk 1]
```diff
@@ -50,6 +51,27 @@
 logger = logs.Logger('builder')  # pylint: disable=invalid-name
 
 
+def get_fuzzer_benchmark_pairs(fuzzers, benchmarks):
+    """Return a tuple of (fuzzer, benchmark) pairs to build. Excludes
+    blacklisted fuzzers from each benchmark.
```
Comment (lszekeres) [inline experiment/build/builder.py:56]: nit: consider using "skip list" instead here and everywhere else.

**Review by lszekeres — APPROVED** [review]

File: experiment/build/builder.py (hunk 1 above)
Comment (inferno-chromium) [reply experiment/build/builder.py:56]: Done!

Comment (lszekeres) [reply experiment/build/builder.py]: thanks!

**Comment by jonathanmetzman** [comment]
Not sure i like this and it feels pretty rushed.

**Comment by inferno-chromium** [comment]
> Not sure i like this and it feels pretty rushed.

Ok, i can either revert or take your time to review. I will hold off on https://github.com/google/fuzzbench/pull/737 as well.

**Pull request merged**